includes/options/class-amp-options-menu.php

```diff
@@ -257,6 +257,16 @@ public function render_validation_handling() {
 					jQuery( '.amp-tree-shaking' ).toggleClass( 'hidden', this.checked && 'native' !== jQuery( 'input[type=radio][name="amp-options[theme_support]"]:checked' ).val() );
 				} ).trigger( 'change' );
 			</script>
+
+			<p>
+				<label for="disable_admin_bar">
+					<input id="disable_admin_bar" type="checkbox" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[disable_admin_bar]' ); ?>" <?php checked( AMP_Options_Manager::get_option( 'disable_admin_bar' ) ); ?>>
```

Comment:
It's great to have the admin bar enabled by default, with this checkbox to disable it. 

Admins might be willing to ignore the AMP validation error of CSS over 50KB, which doesn't even appear for non logged-in users, in order to have the admin bar.